tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/OpenTelemetrySdkTests.cs

```diff
@@ -213,10 +215,11 @@ public async Task IntegrationDisabled(string packageVersion)
         [SkippableTheory]
         [Trait("Category", "EndToEnd")]
         [Trait("RunOnWindows", "True")]
+        [Trait("RequiresDockerDependency", "true")]
```

Comment:
You will need to remove the `[Trait("RunOnWindows", "True")]` so that we do not run it on Windows CI. Theoretically we can run Windows containers in CI, but I don't know that we've done so yet so even if there's a Windows container version of dd-apm-test-agent, this could be unreliable